schwifty/bank_registry/manual_it.json

```diff
@@ -1,666 +1,1122 @@
+		"primary": true,
+		"short_name": "BANCA PASSADORE AND C."
+	},
+	{
+		"bank_code": "03395",
+		"bic": "ITTPIT2M",
+		"country_code": "IT",
+		"name": "ILLIMITY BANK S.P.A.",
+		"primary": true,
+		"short_name": "ILLIMITY BANK"
+	},
+	{
+		"bank_code": "03399",
+		"bic": "EXRRITM2",
+		"country_code": "IT",
+		"name": "<p><b>Bank:</b> EXTRABANCA S.P.A.</p>",
+		"primary": true,
+		"short_name": ""
+	},
+	{
+		"bank_code": "03440",
+		"bic": "BDBDIT22",
+		"country_code": "IT",
+		"name": "BANCO DI DESIO E DELLA BRIANZA",
+		"primary": true,
+		"short_name": "BANCO DI DESIO E DELLA BRIANZA"
+	},
+	{
+		"bank_code": "03441",
+		"bic": "SVTUIT21",
+		"country_code": "IT",
+		"name": "BLU BANCA S.P.A.",
+		"primary": true,
+		"short_name": "BLU BANCA"
+	},
+	{
+		"bank_code": "03442",
+		"bic": "WIDIITMM",
+		"country_code": "IT",
+		"name": "BANCA WIDIBA SPA",
+		"primary": true,
+		"short_name": "BANCA WIDIBA"
+	},
+	{
+		"bank_code": "03475",
+		"bic": "INGBITD1",
+		"country_code": "IT",
+		"name": "ING BANK N.V.",
+		"primary": true,
+		"short_name": "ING BANK"
+	},
+	{
+		"bank_code": "03576",
+		"bic": "BBVAITMM",
+		"country_code": "IT",
+		"name": "BANCO BILBAO VIZCAYA ARGENTARIA S.A.",
+		"primary": true,
+		"short_name": "BANCO BILBAO VIZCAYA ARGENTARIA"
+	},
+	{
+		"bank_code": "03589",
+		"bic": "BKRAITMM",
+		"country_code": "IT",
+		"name": "ALLIANZ BANK FINANCIAL ADVISORS SPA",
+		"primary": true,
+		"short_name": "ALLIANZ BANK"
+	},
+	{
+		"bank_code": "03599",
+		"bic": "CCRTIT2T",
+		"country_code": "IT",
+		"name": "CASSA CENTRALE BANCA - CREDITO COOPERATIVO ITALIANO S.P.A.",
+		"primary": true,
+		"short_name": "CASSA CENTRALE BANCA"
+	},
+	{
+		"bank_code": "03646",
+		"bic": "NTSBITM1",
+		"country_code": "IT",
+		"name": "N26 BANK GMBH",
+		"primary": true,
+		"short_name": "N26 BANK"
+	},
+	{
+		"bank_code": "03665",
+		"bic": "SOBKITM2",
+		"country_code": "IT",
+		"name": "SOLARISBANK AG",
+		"primary": true,
+		"short_name": "SOLARISBANK AG"
+	},
+	{
+		"bank_code": "05034",
+		"bic": "BAPPIT22",
+		"country_code": "IT",
+		"name": "BANCO BPM S.P.A.",
+		"primary": true,
+		"short_name": "BANCO BPM"
+	},
+	{
+		"bank_code": "05036",
+		"bic": "POPRIT3R",
+		"country_code": "IT",
+		"name": "BANCA AGRICOLA POPOLARE DI RAGUSA SOCIETA' COOPERATIVA PER AZIONI",
+		"primary": true,
+		"short_name": "BANCA AGRICOLA POPOLARE DI RAGUSA SOCIETA"
+	},
+	{
+		"bank_code": "05080",
+		"bic": "IMCOIT2A",
+		"country_code": "IT",
+		"name": "BANCA DI IMOLA S.P.A",
+		"primary": true,
+		"short_name": "BANCA DI IMOLA"
+	},
+	{
+		"bank_code": "05116",
+		"bic": "BCVAIT2V",
+		"country_code": "IT",
+		"name": "BANCA VALSABBINA SOCIETA' COOPERATIVA PER AZIONI (IN BREVE LA VALSABBINA)",
+		"primary": true,
+		"short_name": "BANCA VALSABBINA"
+	},
+	{
+		"bank_code": "05142",
+		"bic": "BCPTITNN",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO POPOLARE",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO POPOLARE"
```

Comment:
[low]: I'd keep `BANCA DI CREDITO POPOLARE - TORRE DEL GRECO` from what we got in the funding source's bank name to be more explicit